statsmodels/discrete/tests/results/results_discrete.py

```diff
@@ -37,7 +37,7 @@ def mnlogit_basezero(self):
                 .0229760791, .8443638283, .042138047, .1434089089,
                 .0081338625, .0910979921, .025300888, 1.059954821]
         self.bse = np.reshape(bse, (6,-1), order='F')
-        self.yhat = np.loadtxt(os.path.join(cur_dir,'yhat_mnlogit.csv'))
+        self.linpred = np.loadtxt(os.path.join(cur_dir,'yhat_mnlogit.csv'))
```

Comment:
as long as the git history is getting touched here anyway, mind adding a space after the comma?